packages/headless/src/api/commerce/product-listings/query-listing.http

```diff
@@ -0,0 +1,47 @@
+POST http://localhost:8100/rest/organizations/commercestore/properties/propertyid/commerce/v2/listing
```

Comment:
What's the goal of this file 🤔  ?

Reply:
I was using it to test the API stub. Removed it!